### ISSUE TITLE
Regular HTML comments are removed. (Origin: bugzilla #549093)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#3052]
status NEW severity *normal* in component *build* for *---*
Reported in version *1.5.6* on platform *Other*
Assigned to: Dimitri van Heesch

On 2008-08-23 08:47:01 +0000, Tadeusz Dracz wrote:

    The documentation states:
    
    
    "Finally, to put invisible comments inside comment blocks, HTML style
    comments can be used:
    
    /*!  Visible text */"
    
    The HTML comments are however not present in the output files.
    
    E.g., in the following code, the " text will not output:
    
    /** 
    Description of the class Test
    
     */
    class Test
    {
    }

**Comment by albert-github** [comment]
I think here we have a problem with the interpretation of this feature.
The intention is to have the possibility to place inside comment blocks that doxygen uses (e.g. the `/**` block) some information that is valuable for the developer, but should not appear in the main documentation. This is possible  by means of the mentioned HTML comments.

When having:
```
/// \file

/**
Description of the class Test
<!-- "Comment -->"
 */
class test1
{
};
```
in the main documentation we see (for 1.5.6 as well as for 1.9.0):
> Description of the class Test " 

so the part `<!-- "Comment -->`is removed.

In the source code we still see (when `STRIP_CODE_COMMENTS=NO`) this comment.

I think this problem is usage but the documentation could make this more clear.
I've just pushed a proposed patch, pull request #8299

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by albert-github** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.1.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
